benchmark/metrics_parser.py

```diff
@@ -0,0 +1,276 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+import argparse
+from collections import defaultdict
+import csv
+import itertools
+import json
+
+
+WALL_CLOCK_KEY = u'-- WALL --'
+
```

Comment:
I suppose this is legal, but a bit unusual to have two spaces between what are essentially variable definitions.